attestation-agent/attestation-agent/Cargo.toml

```diff
@@ -58,7 +58,7 @@ kbs = ["kbs_protocol/background_check", "token"]
 # CoCoAS Attestation Token
 coco_as = ["reqwest", "token"]
 
-all-attesters = ["tdx-attester", "sgx-attester", "az-snp-vtpm-attester", "az-tdx-vtpm-attester", "snp-attester"]
+all-attesters = ["tdx-attester", "sgx-attester", "az-snp-vtpm-attester", "az-tdx-vtpm-attester", "snp-attester", "se-attester"]
```

Comment:
I wonder why the CI passes to build the binary as `attester/se-attester` seems to be built only on s390x? https://github.com/confidential-containers/guest-components/blob/main/attestation-agent/attester/Cargo.toml#L57-L58

https://github.com/confidential-containers/guest-components/actions/runs/9546379968/job/26309121829?pr=590#step:9:241

Reply:
se-attester can also be built and run on x86 machine but can only been run on a s390x machine correctly. @Xynnn007

Reply:
It can actually be run on x86 correctly because https://github.com/confidential-containers/guest-components/blob/main/attestation-agent/attester/src/se/mod.rs#L19 returns false on x86 instance.